### DISCUSSION TITLE
build setup now available.

### DISCUSSION THREAD

**Discussion by venkateshpotluri** [make4all/psst#3] [Announcements]
@make4all/sensor-accessibility  I pushed a very basic development workflow and updated the readme with appropriate instructions. Please be sure to pull from main if you are working on code. Please also feel free to suggest or push updates to improve the build infra! I will be adding linting and will eventually setup ci/cd for this repository.

**Comment by venkateshpotluri** [comment]
The linting and testing infra is now checked in! please run yarn test as you write code, and write tests with it. @pelikhan it would be super helpful if you are able to check this and suggest if I should change anything. Please also suggest changes if you think any of this would conflict with eventually using this library with Jacdac-TS.

**Reply by pelikhan** [reply]
Looks good to me !

**Reply by ghost** [reply]
When I execute 'yarn test' I get the output
yarn run v1.22.11
error Command "test" not found.
...

Repro steps:
1. git pull to the current HEAD revision in the main branch (commit 15a3896f)
2. yarn install
3. yarn test

I looked at package.json and I don't see a script defined for the "test" command.  
There is a new script defined for "lint".

I'm new to yarn and Typescript development so I may be missing something obvious.

**Comment by pelikhan** [comment]
To unblock develop, i've turned the library into a small react app with hot reload (reloads on every source change), starting with "yarn start".

Make sure to run "yarn install" again.

This will make it easier to develop the library and eventually we'll clean this up once we're ready to release it.
